lib/ethon/curls/options.rb

```diff
@@ -435,6 +435,8 @@ def self.option_type(type)
       option :easy, :sslengine, :string, 89
       option :easy, :sslengine_default, :none, 90
       option :easy, :sslversion, :enum, 32, [:default, :tlsv1, :sslv2, :sslv3, :tlsv1_0, :tlsv1_1, :tlsv1_2]
+      option :easy, :sslmaxversion, :enum, 32, [:default, :tlsv1, :sslv2, :sslv3, :tlsv1_0, :tlsv1_1, :tlsv1_2]
+      option :easy, :sslminversion, :enum, 32, [:default, :tlsv1, :sslv2, :sslv3, :tlsv1_0, :tlsv1_1, :tlsv1_2]
```

Comment:
@alexeipanov are you sure this works? According to https://curl.se/libcurl/c/CURLOPT_SSLVERSION.html sslversion == sslminversion since version 7.54 and for max you need different "macros" but you just copied line 437 twice. This doesn't seem right.
Would really appreciate if you add a simple test-case especially for the max-version.